07_maintainers_view.md

```diff
@@ -0,0 +1,70 @@
+# Maintainers' Section
```

Comment:
I assume this is a copy or something? If this is the case, it should be removed. Otherwise we'd need to move it somewhere else.

Reply:
What are you talking about? This section is already a part of the current documentation.

Reply:
You have two files called `07_maintainers_view.md` Take a look at the [root of the repo]( https://github.com/TeamNewPipe/documentation/tree/171200ee3cd1cd80f7d3f575cef02f1bd51dbf67). That file doesn't belong there if I am correct.

Reply:
Oh, I see it now.

Reply:
The second one you edited in the `/docs` dir is the correct one.